tests/run_tocgrib2_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# This is a test script for the NCEPLIBS-grib_util project. This tests
+# the tocgrib2 utility.
+#
+# Ed Hartnett, 4/20/24
+
+set -e
+echo ""
+echo "*** Running tocgrib2 tests"
+
+echo "*** Running tocgrib2 without input/output defined. Should return STOP 10"
+unset FORT11
+unset FORT51
+../src/tocgrib2/tocgrib2 && exit 1
+
+echo "*** Running tocgrib2 with input/output defined."
+export FORT11="data/ref_gdaswave.t00z.wcoast.0p16.f000.grib2"
+export FORT51=out.grib2
+../src/tocgrib2/tocgrib2
```

Comment:
Check that an output file was produced.

How shall we confirm it was the correct output? We may have to circle back around after we figure that out, and make this test check results...

Reply:
I'll give you a more detailed update later, but I've yet to be able to successfully produce any output either on my local machine or on the CI.

Once I solve that problem I'll start by just comparing an inventory against a reference file. As for checking if the output is actually correct, I'm not sure.